### DISCUSSION TITLE
Cookies as optional

### DISCUSSION THREAD

**Discussion by scmanjarrez** [acheong08/EdgeGPT#420] [General]
If I'm not mistaken, you're restricted if you don't use cookies (5 messages per conversation). Shouldn't you keep the cookie parameter as optional to avoid this limitation?

Also, do you think implementing history and chat export is feasible? Currently, conversations expire after 6 hours, however the new update allow keeping chats indefinitely and access to them in a "recent activity" space.

**Comment by acheong08** [comment]
Don't upgrade yet. I'll add it back in a more reasonable manner

**Reply by acheong08** [reply]
The implementation of cookies was a bit shit

**Comment by acheong08** [comment]
https://github.com/acheong08/EdgeGPT/releases/tag/0.6.0

**Reply by acheong08** [reply]
Everything is done with a cookies `list[dict]` and the parsing is left to the user.
Expected file/input/parameter
```json
[
	{"name":"_U", "value":"..."},
	{"name":"othershit", "value":"other values"},
]
